wavelets/transform.py

```diff
@@ -253,10 +253,38 @@ def fourier_period(self):
         return getattr(self.wavelet, 'fourier_period')
 
     @property
+    def scale_from_period(self):
+        """Return a function that calculates the wavelet scale
+        from the fourier period
+        """
+        return getattr(self.wavelet, 'scale_from_period')
+
+    @property
     def fourier_periods(self):
         """Return the equivalent fourier periods for the scales used."""
         return self.fourier_period(self.scales)
 
+    @fourier_periods.setter
+    def fourier_periods(self, periods):
+        """Set the scales based on a list of fourier periods"""
+        self.scales = self.wavelet.scale_from_period(periods)
+
+    @property
+    def frequencies(self):
```

Comment:
I would prefer `fourier_frequencies` to avoid confusion with the angular frequencies `w_k`.